src/mrack/providers/openstack.py

```diff
@@ -549,6 +549,12 @@ async def create_server(self, req):
         if specs.get("flavor"):
             del specs["flavor"]
 
+        if "ad" in specs["group"] or "win" in specs["group"] or "win" in specs["image"]:
```

Comment:
Let's use `is_windows_host` method from `util` to have consistent identification of windows hosts all over mrack.  Btw it works bit differently, so the Q is also: "how should we identify win hosts?":
```python
return (
        meta_host.get("os", "").startswith("win")
        or meta_host.get("os_type", "") == "windows"
    )
```

Reply:
Got it.